src/server/api/controllers/messages-controller.js

```diff
@@ -1,10 +1,46 @@
+/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable camelcase */
 const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 const moment = require('moment-timezone');
 
-const getMessages = async () => {
+const getMessages = async (query, channel_id, sender, limit, sort, sort_by) => {
+  const getMes = knex('channel_messages')
```

Comment:
Hmm im not quite sure if you select all messages here or just create the function and then call it when you do the .where method. Do you know?

So what exactly does this method contain? Give a descriptive name of that